kloppy/_providers/signality.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, Union
+
+from kloppy.domain import TrackingDataset
+from kloppy.infra.serializers.tracking.signality import (
+    SignalityDeserializer,
+    SignalityInputs,
+)
+from kloppy.io import FileLike, open_as_file
+
+
+def load(
+    meta_data: FileLike,
+    p1_raw_data: FileLike,
+    p2_raw_data: FileLike,
```

Comment:
This will only work for matches with two phases (i.e., periods). I'm not familiar with Signality data but I assume that a game can have more than two phases if it goes to overtime. I think we should deal with this in the same way as we do for HawkEye data.

Reply:
I've reworked it, so raw tracking data of a match are passed as a list of raw tracking data per period. This way any arbitrary number of periods are supported.